### DISCUSSION TITLE
Act on behalf of a whistleblower tooltip

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3973] [Q&A] [answered]
Buongiorno,
ho notato che il tooltip della funzionalità "_Act on behalf of a whistleblower_" è in lingua inglese anche se il sito è in italiano.
Non ho trovato la voce per tradurla eventualmente dal menu a tendina della sezione **Impostazioni** > **Personalizzazione dei testi**

**Comment by evilaliv3** [comment, marked as answer]
Grazie per aver segnalato questo aspetto.

Questo testo dovrebbe essere automaticamente tradotto dall'applicativo, per cui se così non è verifichiamo e ci accertiamo di correggere in questi giorni nel prossimo rilascio minore.
